### DISCUSSION TITLE
Change attributes of the button by pressing it

### DISCUSSION THREAD

**Discussion by ToniDuque** [MyreMylar/pygame_gui#264] [Q&A]
Someone knows how to change the color of the edge (for example) of the button by pressing it?.

**Comment by MyreMylar** [comment]
You likely want the `_border` colour theming parameters - `selected_border` or `active_border` depending on which behaviour you are after (active is automatically used when you are actually clicking it, selected is a state you can toggle a button into by calling `.select()` and `.unselect()` after clicking on it).

You set these colours in a theme file and they can be set per button with the same ID, per class of buttons (with the same class ID) or for every button, with the most specific type set overriding the less specific.

There is documentation on theming buttons and theme files here:

https://pygame-gui.readthedocs.io/en/latest/theme_guide.html
https://pygame-gui.readthedocs.io/en/latest/theme_reference/theme_button.html
